samples/subsys/ipc/openamp/sample.yaml

```diff
@@ -3,12 +3,7 @@ sample:
   name: OpenAMP example integration
 tests:
   sample.ipc.openamp:
-    platform_allow:
-      - lpcxpresso54114/lpc54114/m4
-      - lpcxpresso55s69/lpc55s69/cpu0
-      - frdm_mcxn947/mcxn947/cpu0
-      - mps2/an521/cpu0
-      - v2m_musca_b1/musca_b1
+    filter: dt_chosen_enabled("zephyr,ipc")
```

Comment:
@hakehuang Just for my understanding, why did you removed the `platform_allow` and added `filter`?
This means that all boards from https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/subsys/ipc/openamp will be tested?

Reply:
> This means that all boards from https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/subsys/ipc/openamp will be tested?
if the dts availabe,  dt_chosen_enabled("zephyr,ipc"), then the openamp shall be runable, if not why they have this node?